corehq/apps/hqwebapp/views.py

```diff
@@ -447,6 +448,12 @@ def domain_login(req, domain, custom_template_name=None, extra_context=None):
     return _login(req, domain, custom_template_name, extra_context)
 
 
+def iframe_login(req, domain):
```

Comment:
Do these views need to be marked location-safe?

Reply:
nit: I think this should be `iframe_domain_login`, as it's a variant of `domain_login`, not `login`

Reply:
4f884724a5bee18b0dd64af9a3c03211721ca6c3

Reply:
and a81f6e211b81bcdab250b8b5303e9e948ae24972